locales/en-US/strings.properties

```diff
@@ -1,29 +1,39 @@
-top_sites=Top Sites
-highlights=Highlights
-visited=Visited
-bookmarked=Bookmarked
-synced=Synced from another device
-open=Open
-topic=Topic
-bookmark=Bookmark
-remove_bookmark=Remove Bookmark
-share=Share
-copy_address=Copy Address
-email_link=Email Link...
-open_new_window=Open in a New Window
-open_private_window=Open in a Private Window
-dismiss=Dismiss
-delete=Delete from History
+header_top_sites=Top Sites
+header_highlights=Highlights
+type_label_visited=Visited
+type_label_bookmarked=Bookmarked
+type_label_synced=Synced from another device
+type_label_open=Open
+type_label_topic=Topic
+menu_action_bookmark=Bookmark
+menu_action_remove_bookmark=Remove Bookmark
+menu_action_share=Share
+menu_action_copy_address=Copy Address
+menu_action_email_link=Email Link…
+menu_action_open_new_window=Open in a New Window
+menu_action_open_private_window=Open in a Private Window
+menu_action_dismiss=Dismiss
+menu_action_delete=Delete from History
 newtab=New Tab
 search_web=Search the Web
-search_for=Search for {search_term} with
+# LOCALIZATION NOTE (search_for_something_with): {search_term} is a placeholder for
+# what the user has typed in the search input field.
+# e.g. 'Search for ' + search_term + 'with:' becomes 'Search for abc with:'
+# The search engine name is displayed as an icon and does not need a translation
+search_for_something_with=Search for {search_term} with:
 search_settings=Change Search Settings
+# LOCALIZATION NOTE (search_header): displayed at the top of the panel
+# showing search suggestions. {search_term} is replaced with the name of the current
+# default search engine. e.g. 'Google Search'
 search_header={search_engine_name} Search
-loading=Loading...
+loading=Loading…
```

Comment:
Oh this was meant as a default prop for Loader.js but I forgot to put a reference to it, will do that!